### DISCUSSION TITLE
PostgreSQL pipeline support?

### DISCUSSION THREAD

**Discussion by banchanKiro** [prisma/prisma#9172] [Q&A] [answered]
PostgreSQL 14 release is scheduled in the third quarter of 2021. And its September.
The feature I am looking forward to in this release is the **Pipeline Mode** that they will be adding.

Is there any plans to add support for this function anytime soon?

**Comment by ryands17** [comment, marked as answer]
A feature request for this [here](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) would be great 🙂

**Reply by banchanKiro** [reply]
Thanks, I will write that up straight away.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
